tests/integration/targets/ndo_synce_interface_policy/tasks/main.yml

```diff
@@ -0,0 +1,266 @@
+# Test code for the MSO modules
+# Copyright: (c) 2024, Anvitha Jain (@anvjain) <anvjain@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI MultiSite host, username and password
+  ansible.builtin.fail:
+    msg: 'Please define the following variables: mso_hostname, mso_username and mso_password.'
+  when: mso_hostname is not defined or mso_username is not defined or mso_password is not defined
+
+# CLEAN ENVIRONMENT
+- name: Set vars
+  ansible.builtin.set_fact:
+    mso_info: &mso_info
+      host: '{{ mso_hostname }}'
+      username: '{{ mso_username }}'
+      password: '{{ mso_password }}'
+      validate_certs: '{{ mso_validate_certs | default(false) }}'
+      use_ssl: '{{ mso_use_ssl | default(true) }}'
+      use_proxy: '{{ mso_use_proxy | default(true) }}'
+      output_level: '{{ mso_output_level | default("info") }}'
+
+# QUERY VERSION
+- name: Query MSO version
+  cisco.mso.mso_version:
+    <<: *mso_info
+    state: query
+  register: version
+
+
+- name: Execute tasks only for MSO version > 4.3
+  when: version.current.version is version('4.3', '>=')
+  block:
+    - name: Remove fabric template
+      cisco.mso.ndo_template: &template_absent
+        <<: *mso_info
+        name: ansible_fabric_policy_template
+        type: fabric_policy
+        state: absent
+
+    - name: Create a fabric template
+      cisco.mso.ndo_template:
+        <<: *mso_info
+        name: ansible_fabric_policy_template
+        type: fabric_policy
+        state: present
+
+    # CREATE
+    - name: Create a new syncE interface policy (check mode)
+      cisco.mso.ndo_synce_interface_policy: &add_synce_interface_policy
+        <<: *mso_info
+        template: ansible_fabric_policy_template
+        interface_policy: ansible_test_synce_interface_policy
+        state: present
+      check_mode: true
+      register: cm_add_synce_interface_policy
```

Comment:
Can you add the proposed value check for the check mode tests?

Reply:
Proposed value check will not be added now, since the existing modules with proposed gives the below error 
----> 'dict object' has no attribute 'proposed'

This will be fixed later.